vulcanai/models/metrics.py

```diff
@@ -427,27 +451,25 @@ def get_auc(targets, raw_predictions, num_classes, average=None):
 
         all_class_auc = []
         for i in range(num_classes):
-            if num_classes == 1:
-                fpr, tpr, _ = skl_metrics.roc_curve(targets,
-                                                    raw_predictions,
-                                                    pos_label=1)
-            else:
-                fpr, tpr, _ = skl_metrics.roc_curve(targets,
+            fpr, tpr, _ = skl_metrics.roc_curve(targets,
```

Comment:
Since we technically could be getting two forms of binary inputs (e.g. [[0, 1],[1, 0]] and [1, 0]), this was to handle those 2 cases. @nirtiac would the pos_label generalize to these cases? I would love for us to be able to condense it but I want to be absolutely sure that it's still fine before we do that.

Reply:
ahhhhh no it doesn't necessarily..... will revert. have been testing with just the one case.